New post: T2T human genome (#931)
usegalaxy-eu/website: _posts/2022-04-06-T2T-human-genome.md

---
site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]

> title: 'T2T-CHM13v2.0 available in Galaxy'

Reviewer: The title should probably more telling. People usually do not know the acronym. Complete human genome () available.... Or something like that?

Author: > I think the core part regarding Galaxy is still lacking. I would add a final paragraph on where/how that genome is now available (for which tools/from which datatables)?

Sure, I'll add it.

---

> Despite the first version of the human genome generated as a result of the Telomere-to-Telomere (T2T) Consortium efforts dates back to 2020, it has taken two years of additional work to achieve something that the scientific community was eagerly awaiting: the first complete, gap-free sequence for all 24 human chromosomes.

Reviewer 2: ```suggestion
While the first version of an improved human genome resulting from the Telomere-to-Telomere (T2T) Consortium efforts got reported in 2020, it has taken two years of additional work to achieve something that the scientific community has been eagerly awaiting since: the first complete, gap-free sequence for all 24 human chromosomes (including Y).

```

Reviewer 2: Do you have a citation for their version 1?

Author: I took as reference the first entry in NCBI (the version 0.7 https://www.ncbi.nlm.nih.gov/assembly/GCA_009914755.1/)

> Despite the first version of the human genome generated as a result of the Telomere-to-Telomere (T2T) Consortium effords dates back 2020, it has taken two years of additional work to achieve something that the scientific community was eagerly awaiting: the first complete, gap-free sequence for all 24 human chromosomes has been published.

Reviewer 3: ```suggestion
Despite the first version of the human genome generated as a result of the Telomere-to-Telomere (T2T) Consortium efforts dates back to 2020, it has taken two years of additional work to achieve something that the scientific community was eagerly awaiting: the first complete, gap-free sequence for all 24 human chromosomes.
```

<br>

![Graphical Abstract](/assets/media/T2T_human_genome.png)

*Resolved sequences by the T2T-CHM13v2.0 reference genome. Resource: T2T consortium*

<br>
 
    

> The T2T-CHM13v2.0 reference genome, generated primarily by long-read sequencing, provides a significant improvement in the characterization of the centromeric satellite repeats, transposable elements, and segmental duplications.

Reviewer 2: ```suggestion
The T2T-CHM13v2.0 reference genome, generated primarily by long-read sequencing, provides a significant improvement in the characterization of centromeric satellite repeats, transposable elements, and segmental duplications.
```

> The T2T-CHM13v2.0 reference genome, generated primarily by long-read sequencing, provides a signiticative improvement in the characterization of the centromeric satellite repeats, transposable elements, and segmental duplications.

Reviewer 3: ```suggestion
The T2T-CHM13v2.0 reference genome, generated primarily by long-read sequencing, provides a significative improvement in the characterization of the centromeric satellite repeats, transposable elements, and segmental duplications.
```

Reviewer 2: or "significant" maybe ;)

Author: Yes, perhaps "significant" sounds better.

> Compared to the latest version of the human reference genome GRCh38, it adds nearly 200 million base pairs of novel DNA sequences, revealing 2,880 genes with not assined ortologs in GRCh38 and nearly 2,000 candidate new genes. The filled gaps include the entire short arms of five human chromosomes and cover some of the most complex regions of the genome. In addition, it corrects thousands of structural errors, including the complete structure of the Y chromosome for the first time.

Reviewer 2: ```suggestion
Compared to the latest version, GRCh38, of the human reference genome, it adds nearly 200 million base pairs of novel DNA sequences, revealing 2,880 genes with no assigned GRCh38 orthologs, and nearly 2,000 candidate new genes. The filled gaps include the entire short arms of five human chromosomes and cover some of the most complex regions of the genome. In addition, the new version corrects thousands of structural errors and includes, for the first time, the complete sequence of the Y chromosome.
```

> Compared to the latest version of the human reference genome GRCh38, it adds nearly 200 million base pairs of novel DNA sequences, revealing 2880 genes with not assined ortologs in GRCh38 and nearly 2,000 candidate new genes. The filled gaps include the entire short arms of five human chromosomes and cover some of the most complex regions of the genome. In addition, it corrects thousands of structural errors, including the complete structure of the Y chromosome for the first time.

Reviewer 3: ```suggestion
Compared to the latest version of the human reference genome GRCh38, it adds nearly 200 million base pairs of novel DNA sequences, revealing 2,880 genes with not assined ortologs in GRCh38 and nearly 2,000 candidate new genes. The filled gaps include the entire short arms of five human chromosomes and cover some of the most complex regions of the genome. In addition, it corrects thousands of structural errors, including the complete structure of the Y chromosome for the first time.
```
